examples/src/main/java/com/cloudera/nav/plugin/client/examples/updatedResults/GetAllUpdatedResults.java

```diff
@@ -0,0 +1,32 @@
+package com.cloudera.nav.plugin.client.examples.updatedResults;
+
+import com.cloudera.nav.plugin.client.*;
```

Comment:
Please see the Cloudera enterprise Java style guide for guidance regarding imports. You should set these up as preferences in IntelliJ. I can show you my settings